src/sentry/eventstore/base.py

```diff
@@ -272,3 +272,30 @@ def bind_nodes(self, object_list, node_name="data"):
             for item, node in object_node_list:
                 data = node_results.get(node.id) or {}
                 node.bind_data(data, ref=node.get_ref(item))
+
+    def get_unfetched_transactions(
+        self,
+        snuba_filter,
+        orderby=None,
+        limit=100,
+        offset=0,
+        referrer="eventstore.get_unfetched_transactions",
+    ):
+        """
+        Same as get_unfetched_events but returns transactions.
+        Only the event ID, projectID, groupID and timestamp field will be present without
```

Comment:
transactions don't have `groupID` right? I'm assuming this just left over from the copy pasta

Reply:
Done